rules/pandoc-citeproc.json

```diff
@@ -6,7 +6,8 @@
         "constraints": [
           {
             "os": "linux",
-            "distribution": "ubuntu"
+            "distribution": "ubuntu",
+            "versions": ["20.04", "22.04"]
```

Comment:
I wonder if you'd like to add the older versions here as well. Otherwise this is a breaking change for them.

Reply:
Thanks for reviewing this. Ubuntu 18 and older are no longer supported, but might as well I guess. There are almost definitely some Ubuntu 18 usages still out there.

Reply:
Yeah, I meant for reproducibility. New sysreqs entries do not need to support them of course, but it would be great if the behavior for the already existing ones didn't change. Thanks for adding them!